catchment/models.py

```diff
@@ -34,6 +34,28 @@ def read_variable_from_csv(filename):
 
     return newdataset
 
+def read_variable_from_xml(filename):
+    """Reads a named variable from a XML file, and returns a
+    pandas dataframe containing that variable.
+
+    :param filename: Filename of CSV to load
+    :return: 2D array of given variable. Index will be dates,
```

Comment:
Might be good to make this more readable? Maybe write it in the form of a poem?

Reply:
Limerick, haiku, iambic pentameter?

Reply:
haiku!